### PR TITLE
PR : For deliver task 3 arm sprint led control v2.0

### PR DESCRIPTION
**Pull request by Khedr05**

### CODE REVIEW

File: 07_Sprint_Twelve/02_RGB_LED_CONTROL_V2.0/00_Project/RGB_LED_SEQUANCE_V2.0/APPLICATION/app.c [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+				for(VUint8_t i=0;i<200;i++)
+				{
+					for(VUint8_t j =0 ; j<100;j++);
+				}
+			}
+		
+			 pressCounter++;
+		}
+		
+		if(pressCounter == 1)
```
Comment (Hala-Abdelkader) [inline 07_Sprint_Twelve/02_RGB_LED_CONTROL_V2.0/00_Project/RGB_LED_SEQUANCE_V2.0/APPLICATION/app.c]: You shouldn't use magic numbers

**Review by Hala-Abdelkader — CHANGES_REQUESTED** [review]

File: 07_Sprint_Twelve/02_RGB_LED_CONTROL_V2.0/00_Project/RGB_LED_SEQUANCE_V2.0/APPLICATION/app.c (hunk 1 above)
Comment (Khedr05) [reply 07_Sprint_Twelve/02_RGB_LED_CONTROL_V2.0/00_Project/RGB_LED_SEQUANCE_V2.0/APPLICATION/app.c]: Done Now You Can Check The Last Commit

**Comment by Khedr05** [comment]
Done Now You Can Check The Last Commit

File: 07_Sprint_Twelve/02_RGB_LED_CONTROL_V2.0/00_Project/RGB_LED_SEQUANCE_V2.0/APPLICATION/app.c (hunk 1 above)
Comment (Hala-Abdelkader) [reply 07_Sprint_Twelve/02_RGB_LED_CONTROL_V2.0/00_Project/RGB_LED_SEQUANCE_V2.0/APPLICATION/app.c]: Great, well done.

**Pull request merged**